fitter/probabilities/pulsars.py

```diff
@@ -357,16 +336,20 @@ def cluster_component(model, R, mass_bin, DM=None, ΔDM=None, DM_mdata=None, *,
         Paz_dist /= 2
 
     # Change the acceleration domain to a Pdot / P domain
+    # TODO (Peter): Reminder here to double check with Nolan/Vincent that this
+    # final distribution should be normalized to 1.0, even after the conversion
+    # from az to Pdot/P.
```

Comment:
I think we determined it shouldn't really matter eh?

Reply:
Yep, fixed it regardless, I'll remove this todo when I go through and make the other changes.